charts/ironic-standalone/README.md

```diff
@@ -0,0 +1,168 @@
+-----------------------------------------
+
+Tftp works only with service_type=ClusterIP with ExternalIPs managed by keepalived.
+If your k8s cluster uses metallb with layer2, tftp won't work.
+With metallb, best option is to host tftp service outside of the k8s cluster.
+
+Breaking change in 1.0.0
+------------------------
+The chart switches from MySQL to Mariadb DB engine.
+Create a SQL dump on existing database before upgrade and load it to the newly created Mariadb.
+Note that configuration parameters for databases change.
+
+Overview
+--------
+
+[![Build Status](https://travis-ci.org/jakub-d/openstack-ironic-standalone.svg?branch=master)](https://travis-ci.org/jakub-d/openstack-ironic-standalone)
+
+[Openstack Ironic](https://docs.openstack.org/ironic/latest/index.html), Ironic is an OpenStack project which provisions bare metal (as opposed to virtual) machines. It may be used independently or as part of an OpenStack Cloud, and integrates with the OpenStack Identity (keystone), Compute (nova), Network (neutron), Image (glance), and Object (swift) services.
+
+This Helm chart installs Openstack Ironic in [standalone mode](https://docs.openstack.org/ironic/latest/install/standalone.html).
+
+TL;DR;
+
+Create a config file (`my-site.yaml`) with following variables defined:
+```
+---
+ironicServerName: example.example.io
+persistentVolumeClaimName: existing-claim-name
+api:
+  externalIPs:
+    - 10.10.10.10
+httpboot:
+  externalIPs:
+    - 10.10.10.10
+tftp:
+  externalIPs:
+    - 10.10.10.10
+mariadb:
+  db:
+    password: secret1
+  rootUser:
+    password: secret1
+  master:
+    persistence:
+      existingClaim: existing-claim-name
+rabbitmq:
+  rabbitmq:
+    password: secret3
+```
+(you don't have to define secrets, but this step is required for production, as random-generated secrets will change each helm roll-over and will not match persistent storage db)
+
+Another example of the config file, using ingress and dynamic pv provisioners (make sure the tftp storageClass used allows cross-pod mounting):
+```
+---
+ironicServerName: ironic.example.io
+api:
+  ingress:
+    enabled: true
+    hosts:
+      - ironic.example.io
+httpboot:
+  ingress:
+    enabled: true
+    hosts:
+      - ironicwww.example.io
+tftp:
+  externalIPs:
+    - 10.10.10.10
+  nodeSelector:
+    hostname: 10.10.10.10.xip.io
+  persistence:
+    storageClass: sharefs
+mysql:
+  mysqlPassword: secret1
+  mysqlRootPassword: secret2
+rabbitmq:
+  rabbitmq:
+    password: secret3
+```
+
+Then install a chart using command:
+```
+helm repo add ironic https://ironic.storage.googleapis.com
+helm install ironic/openstack-ironic-standalone -f my-site.yaml
+```
+
+Keystone Authentication (Optional)
+==================================
+
+This chart deploys only Openstack Ironic by default.
+You can enable Keystone service by providing extra configuration parameters in the deployment:
+
+```
+keystone:
+  enabled: true
+  admin_password: ADMIN-PASSWORD
```

Comment:
This looks like a pretty dated keystone configuration. I think (not sure) admin_password is no longer a thing.

Reply:
This is keystone configuration, not basic auth.